packages/native/src/components/Layout/List/List/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components/native";
+import Text from "../../../Text";
+import { Box, Flex } from "../../index";
+import baseStyled, { BaseStyledProps } from "../../../styled";
+import { FlatList } from "react-native";
+
+export type BaseListItemProps = {
+  title?: string;
+  description?: string;
+  bullet?: React.ReactNode;
+};
+
+export type BaseListProps = {
+  items: BaseListItemProps[];
+  itemContainerProps?: BaseStyledProps;
+};
+
+const ListContainer = styled.View`
+  display: flex;
+  flex-direction: column;
+`;
+
+const ListItemContainer = baseStyled.View`
+  display: flex;
+  flex-direction: row;
+`;
+
+export const ListItem = ({
+  title,
+  description,
+  bullet = null,
+  ...props
+}: BaseStyledProps & BaseListItemProps): React.ReactElement => {
+  return (
+    <ListItemContainer {...props}>
```

Comment:
What about setting `alignItems="center"` by default? I think this is what will be used 90% of the time.

Reply:
Well i don't know... in the onboarding figma, sometimes it's alignItems "center" (when the item height is one line), sometimes it's "start"...

Reply:
However alignItems="center" looks good in most case i will set it on default yes